# Client revision number
uint64 rev_number
---
# World mode update (delta)
WorldModelDelta world

# New revision number
uint64 rev_number

# Error message. Is empty if no errors occurred
string error
